modules/hydrodyn/python-lib/hydrodyn_library.py

```diff
@@ -421,6 +421,7 @@ def check_error(self):
     def check_input_motions(self,nodePos,nodeVel,nodeAcc):
         # make sure number of nodes didn't change for some reason
         if self._initNumNodePts != self.numNodePts:
+            # @ANDY TODO: `time` is not available here so this would be a runtime error
```

Comment:
Good catch.  This error never occurred in the test cases, so it never got triggered.